src/transformers/models/pegasus/configuration_pegasus.py

```diff
@@ -167,3 +172,227 @@ def num_attention_heads(self) -> int:
     @property
     def hidden_size(self) -> int:
         return self.d_model
+
+
+# Copied from transformers.models.bart.configuration_bart.BartOnnxConfig with Bart->Pegasus
+class PegasusOnnxConfig(OnnxSeq2SeqConfigWithPast):
```

Comment:
You can deal with the tolerance issue by adding the following property to this class (you could try with 1e-4 first and lower to 1e-3 if needed):

```
    @property
    def atol_for_validation(self) -> float:
        return 1e-4
```

Reply:
@lewtun sorry, just got to checking on this. I'm seeing the below when I run the tests. Does the **F** mean failed? I tried with both 1e-3 and 1e-4 and I see that if I don't use the RUN_SLOW flag the tests seem to succeed.
<img width="776" alt="image" src="https://user-images.githubusercontent.com/16939722/200143087-6787ef11-a3ff-4884-ac42-9cc81e322b98.png">